x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/combine_renderers/index.tsx

```diff
@@ -33,9 +34,13 @@ export const combineRenderers = ({
     isDraggable: boolean;
     scopeId: string;
   }) => (
-    <>
-      {a.isInstance(data) && a.renderRow({ contextId, data, isDraggable, scopeId })}
-      {b.isInstance(data) && b.renderRow({ contextId, data, isDraggable, scopeId })}
-    </>
+    <EuiFlexGroup direction="column" gutterSize="m">
```

Comment:
better spacing for Event Renderers.

### Before

![grafik](https://github.com/user-attachments/assets/c214a60f-fef3-4756-81b0-97e91d094923)


### After

![grafik](https://github.com/user-attachments/assets/8aa95dea-299f-4b1f-b5f0-de91eeac8139)